app/src/bcwallet-theme/features/person-flow/utils/BCIDHelper.ts

```diff
@@ -115,15 +121,21 @@ export const cleanupAfterServiceCardAuthentication = (status: AuthenticationResu
   }
 }
 
-export const initiateAppToAppFlow = async (url: string) => {
+export const initiateAppToAppFlow = async (
+  url: string,
+  t: TFunction<'translation', undefined>,
+  logger?: BifoldLogger
+) => {
   try {
     if (await Linking.canOpenURL(url)) {
       await Linking.openURL(url)
     } else {
       throw new Error()
     }
-  } catch {
-    const error = new BifoldError('Error.Title2032', 'Error.Message2032', 'Error.NoMessage', 2032)
+  } catch (err: unknown) {
+    logger?.error(`Error opening URL ${(err as Error).message}`)
```

Comment:
```suggestion
    logger?.error(`Error opening URL ${(err as Error)?.message}`)
```

Just so if err happens to be undefined for whatever reason it doesn't break things